node/src/chain_spec.rs

```diff
@@ -72,6 +73,20 @@ pub fn development_config() -> Result<ChainSpec, String> {
 				"0xec00ad0ec6eeb271a9689888f644d9262016a26a25314ff4ff5d756404c44112"
 					.parse()
 					.unwrap(),
+				// SnowBridged Assets
+				vec![
+						(
+							"Mangata".as_bytes().to_vec(), "MNG".as_bytes().to_vec(), "Mangata Asset".as_bytes().to_vec(),
+							18u32, 0u32, H160::from_slice(&hex!["F8F7758FbcEfd546eAEff7dE24AFf666B6228e73"][..]),
+							1_000_000u128, get_account_id_from_seed::<sr25519::Public>("Alice")
```

Comment:
Let there be 100 million Mangata tokens.
Shouldn't the total supply be in the smallest denomination?

Reply:
Do you mean "u128 -> u32"?
I used u128 because u128 is what is set as Balance in runtime/lib.rs

Reply:
Here I mean the amount is set not by specifying the number of whole tokens, but by the amount of smallest, indivisible shares of it. So, if a token has 6 numbers after a decimal point and you want to have 1 token in total, you should specify 1_000_000 here, similar as you do in ethereum smartcontracts. Am I wrong?

Reply:
Oh right! I completely missed that!
Thank you for pointing that out.